### PR TITLE
Sync metallb operator 200624

### PR DESCRIPTION
**Pull request by karampok**
There were some conficts in docker file, and I did go vendor/go mod tidy on top

### CODE REVIEW

**Comment by karampok** [comment]
Is there something blocking from being merged? (labels are not applied I think)

File: manifests/stable/metallb.io_bgppeers.yaml [hunk 1]
```diff
@@ -6,6 +6,18 @@ metadata:
   creationTimestamp: null
   name: bgppeers.metallb.io
 spec:
+  conversion:
+    strategy: Webhook
```
Comment (karampok) [inline manifests/stable/metallb.io_bgppeers.yaml:10]: @fedepaol  not sure if that is needed ds

**Comment by karampok** [comment]
If we can merge that, then the last commit will be reset after succesfully manage the metallb repo (the FRR params test remains to be fixed though)

**Comment by fedepaol** [comment]
> If we can merge that, then the last commit will be reset after succesfully manage the metallb repo (the FRR params test remains to be fixed though so the metallb ci to be green)

I'd prefer to do things cleanly

**Comment by karampok** [comment]
Updated with no extra commit that does not exist in upstream (apart from the go stuff)

**Comment by karampok** [comment]
@fedepaol  ci is green as expected. Needs lgtm, thanks

**Pull request merged**